src/main/java/pl/touk/controller/DriverController.java

```diff
@@ -0,0 +1,31 @@
+package pl.touk.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.touk.service.DriverService;
+import pl.touk.service.dto.DriverDto;
+import pl.touk.service.dto.NewDriverDto;
+import pl.touk.service.dto.UpdateDriverDto;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class DriverController {
+
+    private final DriverService driverService;
+
+    @PostMapping("/newDriver")
+    public NewDriverDto startParkingMeter(@RequestBody DriverDto driverDto) {
+        log.info("Insert object [ " + driverDto.getRegistrationNumber() + "]");
+        return driverService.startParkingMeter(driverDto.getRegistrationNumber());
+    }
+
+    @PostMapping("/updateDriver")
```

Comment:
1) Czy to jest ścieżka zgodna z RESTem?
2) Czy POST to na pewno dobra metoda?

Reply:
1) Nie jest. powinno być @PostMapping("/update")
2) Wg mnie tak. Zastosowanie metody PUT czy POST jest kwestią umowną, różne są też opinie na stackOverflow na ich temat. Nie znalazłem znaczących różnic technicznych między tymi dwoma metodami.